cfgov/jinja2/v1/events/_macros.html

```diff
@@ -236,10 +236,12 @@ <h2 class="event-venue_heading">
               block__padded-top
               block__border-top">
     <h2>Agenda</h2>
-    <table class="u-w100pct
+    <table class="o-table
+                  table__stack-on-small
+                  table__entry-header-on-small
```

Comment:
Should the event agenda table have the `stack-on-small` class? It looks like it conflicts with the `u-w25pct` on the table cells at small sizes:

<img width="371" alt="screen shot 2016-08-15 at 12 04 39 pm" src="https://cloud.githubusercontent.com/assets/778171/17676402/3fe87cc8-62e3-11e6-9823-ca18fdbcdb07.png">